### ISSUE TITLE
No Announcement of Search Results for Non-Visual Users

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1778]
### Steps to Reproduce

1. Navigate to My Catalog page in admin interface.
2. Navigate to search field, type query, execute search
3. Notice that the page is refreshed and focus is returned to the start of the page.
4. Notice that there is no aria-live or equivalent event to notify non visual users of the results of their search
5. Use the keyboard to navigate to the search results, notice that this takes a significant time, even when using an assistive technology like a screen reader.

**Expected behavior:** 
Assistive tech users should be given an update about the results of their search. Visual users can tell immediately if their search returned results while non-visual users have to browse through the entire page to see if the results table is populated. It is recommended that an aria-live region notifies users with a message like "Search returned 4 items" or "Search return no items".
**Actual behavior:** 
Currently non-visual users are given no notification and must navigate to the table of results to find out if their search was successful.

**Comment by connerbw** [comment]
[Done like in WordPress](https://github.com/WordPress/wordpress-develop/blob/669cefe58723b3d5c7912282b94f41ea5b2c236c/src/wp-admin/network/themes.php#L276-L279). Added results totals. 

![Search Results](https://user-images.githubusercontent.com/812192/67977992-7d552b80-fbef-11e9-8a7c-ffbaf59b7a2f.png)

**Comment by avaaec** [comment]
This is a UW ATS priority 2 issue

**Comment by SteelWagstaff** [comment]
UW reports that fix does not produce expected result. See screen recording of issue: https://drive.google.com/file/d/13K08oJltTGgluZLHFob7CHmbG8KWPCgA/view

**Comment by SteelWagstaff** [comment]
We should also look at where focus is placed following submission of a search query.

**Comment by greatislander** [comment]
As described in the [ARIA22](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22) technique:

> This technique uses the `status` role from the ARIA specification to notify Assistive Technologies (AT) when content has been updated with information about the user's or application's status.

One of the examples in this technique specifically talks about search results:

> After a user presses a Search button, the page content is updated to include the results of the search, which are displayed in a section below the Search button. The change to content also includes the message "5 results returned" near the top of this new content. This text is given an appropriate role for a status message. A screen reader will announce "Five results returned".

However, this example references a situation where the current page is _updated_ with new information. See also the MDN Web Docs article on [ARIA live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions):

> Using JavaScript, it is possible to dynamically change parts of a page without requiring the entire page to reload — for instance, to update a list of search results on the fly, or to display a discreet alert or notification which does not require user interaction. While these changes are usually visually apparent to users who can see the page, they may not be obvious to users of assistive technologies. ARIA live regions fill this gap and provide a way to programmatically expose dynamic content changes in a way that can be announced by assistive technologies.

However, the search in Pressbooks described in this issue is performed using a form submission which loads a _new_ page with results, announcing the page load to AT users. I'm not certain this meets requires that the change be announced.

As per the discussion [here](https://github.com/w3c/aria/issues/1360), it seems that `role="alert"` is not consistently supported by screen readers, which means the only way to achieve this is by inserting content into the wrapper element using JavaScript once the page has loaded. I've implemented this in https://github.com/pressbooks/pressbooks/pull/2139. However, I'm not convinced this is an accessibility issue; note that the _current_ behaviour in Pressbooks matches that on the W3C Web Accessibility Initiative page: https://www.w3.org/WAI/search/?q=aria
